README.md

```diff
@@ -173,6 +174,16 @@ Kind: *Trusted Indication*.
 
 Errors:
 
+### `AccountClaim`
+Sends a claim verified to be from the `source` to the `destination` on the parachain. It has an `expiration` timestamp and a `scope` used to define the scope under which such a `claim` is made. A chain should only send such a claim if it is verified to have been intended to be sent by the `source`. It facilitates authentication across parachains.
+
+- `source: MultiLocation`
```

Comment:
how does this differ from `origin`?

Reply:
Various, let's assume you use a claim to authorize an account to execute trades on behalf of it one a target chain, you could use the scope to limit it to a certain currency.

Reply:
I'll remove source as it is indeed the same as `origin`